lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers.go

```diff
@@ -127,6 +128,23 @@ type subscriberFilter func(sub *subscribermodels.Subscriber) bool
 
 func acceptAll(*subscribermodels.Subscriber) bool { return true }
 
+// filterSubscribersIfNonverbose filters a subscribers list for the specified verbosity.
+// If verbose mode is used, the list is returned as-is (a map of MSISDN strings
+// to Subscribers).
+// If non-verbose mode is used, an array of MSISDN strings is returned instead.
+func filterSubscribersIfNonverbose(subs map[string]*subscribermodels.Subscriber, verbose bool) interface{} {
```

Comment:
Small request: I think "filter" is a misnomer here, because we're actually doing a conditional *map*, rather than *filter*. I'd go with `mapVerbose` or `mapSubscribersForVerbosity` or something like that